packages/core/src/components/forms/numericInput.tsx

```diff
@@ -236,8 +246,11 @@ export class NumericInput extends AbstractComponent<HTMLInputProps & INumericInp
             const decrementButton = this.renderButton(
                 NumericInput.DECREMENT_KEY, NumericInput.DECREMENT_ICON_NAME, this.handleDecrementButtonClick);
 
+            const buttonGroupClasses = classNames(Classes.BUTTON_GROUP, Classes.VERTICAL, Classes.FIXED, {
+                [Classes.LARGE]: large,
```

Comment:
👎 
but not actually! what's providing the style there is `.pt-large .pt-button` (button inside any large container). the descendant selector was my clever trick to not have to reimplement large buttons on any possible container.